### PR TITLE
Using check-manifest to update manifest

### PR DESCRIPTION
**Pull request by daviddavis**
refs #7656

Please be sure you have read our documentation on creating PRs:
https://docs.pulpproject.org/contributing/pull-request-walkthrough.html

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -67,3 +67,23 @@ exclude = '''
...
+
+[tool.check-manifest]
+ignore = [
+    ".bumpversion.cfg",
+    "CHANGES/**",
+    "CONTRIBUTING.rst",
+    "HISTORY.rst",
+    "dev_requirements.txt",
+    "doc_requirements.txt",
+    "docs/**",
```
Comment (bmbouter) [inline pyproject.toml:79]: Does this mean we're not including the docs in our shipped python wheels? Or are they still included in the sdists? Do we publish sdists or wheels only?

Comment (daviddavis) [reply pyproject.toml:79]: I'm hoping someone with more knowledge of our packaging weighs in here but I believe we aren't shipping docs with our wheels. Not sure about your other questions.

**Review by goosemania — APPROVED** [review]

File: pyproject.toml (hunk 1 above)
Comment (goosemania) [reply pyproject.toml:79]: FWIW, [MANIFEST.in affects sdists only](https://packaging.python.org/guides/distributing-packages-using-setuptools/#manifest-in), not bdists like wheels. So I believe docs are not included in the sdists.
Not sure what is included in to bdist by default.

Comment (bmbouter) [reply pyproject.toml:79]: Great, thank you.

**Review by bmbouter — APPROVED** [review]
Thanks!

**Pull request merged**